src/runtime_src/core/edge/user/aie/common_layer/adf_aie_control_api.h

```diff
@@ -0,0 +1,95 @@
+/**
+* Copyright (C) 2021 Xilinx, Inc
```

Comment:
Update to 2022?

Reply:
Will update it using another pull request